scst-store/deployment-details.hbs.md

```diff
@@ -6,23 +6,27 @@ This topic describes how you can deploy and configure your Kubernetes cluster fo
 
 The installation creates the following in your Kubernetes cluster:
 
-- Two components — an API back end and a database.
-  Each component includes:
-    - service
-    - deployment
-    - replicaset
-    - Pod
-- Persistent volume claim
-- External IP address (based on a deployment configuration set to use `LoadBalancer`).
-- A Kubernetes secret to allow pulling SCST - Store images from a registry.
+- Four components:
+    - metadata-store API backend 
+    - database
+    - AMR API backend (If AMR is deployed, see [Deploying AMR](#amr))
+    - AMR persister (If AMR is deployed, see [Deploying AMR](#amr))
```

Comment:
Can we change "AMR persister" to "AMR CloudEvent Handler" ? That's how we're referring to it in the observer docs (and I think how we want to refer to it externally)

Reply:
made the change!